index.js

```diff
@@ -313,6 +315,17 @@ function Argv (processArgs, cwd) {
     return options
   }
 
+  var groups = {'Options:': []}
```

Comment:
Any worry about duplicating/defining the `'Options:'` string in both index.js and lib/usage.js?